### PR TITLE
Convert integer to its English words representation

### PR DESCRIPTION
**Pull request by ann-kolosova**
Convert a non-negative integer num to its English words representation.

### CODE REVIEW

File: Lesson07_Task01.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    '9': 'Ninety',
+}
+
+if 100 <= num <= 999:
+    if digit[1] == '0' and digit[2] == '0':
+        print(f'"{hundreds_ones[digit[0]]} Hundred"')
+    elif digit[1] == '1':
+        print(f'"{hundreds_ones[digit[0]]} Hundred {tens_1[digit[1:]]}"')
+    elif digit[1] == '0' and digit[2] != '0':
+        print(f'"{hundreds_ones[digit[0]]} Hundred {hundreds_ones[digit[0]]}"')
```
Comment (antohaUa) [inline Lesson07_Task01.py:63]: Here should be ( check your initial solution vs "108". I hope it was typo, but please be careful next time and check your solutions with different data sets.
` f'"{hundreds_ones[digit[0]]} Hundred {hundreds_ones[digit[2]]}"'`

**Review by antohaUa — APPROVED** [review]

File: Lesson07_Task01.py (hunk 1 above)
Comment (ann-kolosova) [reply Lesson07_Task01.py]: fixed and checked with the next data: '108', '130'

File: Lesson07_Task01.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    '8': 'Eighty',
+    '9': 'Ninety',
+}
+
+if 100 <= num <= 999:
+    if digit[1] == '0' and digit[2] == '0':
+        print(f'"{hundreds_ones[digit[0]]} Hundred"')
+    elif digit[1] == '1':
+        print(f'"{hundreds_ones[digit[0]]} Hundred {tens_1[digit[1:]]}"')
+    elif digit[1] > '0' and digit[2] == '0':
```
Comment (antohaUa) [inline Lesson07_Task01.py:62]: `digit[1] > '0'`  is redundant cause case when digit[1] == 0 covered here `if digit[1] == '0' and digit[2] == '0':`

**Review by antohaUa — APPROVED** [review]

File: Lesson07_Task01.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    '5': 'Fifty',
+    '6': 'Sixty',
+    '7': 'Seventy',
+    '8': 'Eighty',
+    '9': 'Ninety',
+}
+
+if 100 <= num <= 999:
+    if digit[1] == '0' and digit[2] == '0':
+        print(f'"{hundreds_ones[digit[0]]} Hundred"')
```
[the remark this answers is not in the document]
Comment (ann-kolosova) [reply Lesson07_Task01.py]: fixed with .join method

**Pull request merged**